src/devices/cpu/mpk1839/kl1839vm1.h

```diff
@@ -75,17 +95,24 @@ class kl1839vm1_device :  public cpu_device
 	memory_access<24, 0, 0, ENDIANNESS_BIG>::specific m_sysram;
 	memory_access<24, 0, 0, ENDIANNESS_LITTLE>::specific m_ram;
 	memory_access<6, 2, -2, ENDIANNESS_LITTLE>::specific m_io;
-	PAIR                m_mca;
-	PAIR                m_vma;
-	PAIR                m_vma_tmp; // does we have int reg for this?
-	PAIR                m_rv;
-	PAIR                m_sch;
-	PAIR                m_rsp;
-	PAIR                m_ppc;    // previous program counter
-	bool                m_fp;
-	u32                 m_consts[0x10] = { 0x4, 0x2, 0x8, 0x1, 0x0, 0, 0, 0x66, 0, 0xc00000, 0xffffffff, 0x1f0000, 0x4000000, 0, 0, 0 };
-	PAIR                m_reg[0x20];
-	int                 m_icount;
+
+	std::unique_ptr<util::disasm_interface> m_vax_dasm;
+
+	PAIR            m_vma_tmp; // do we have int reg for this?
+	PAIR            m_rv;
+	PAIR            m_sch;
+	PAIR            m_rsp;
+	PAIR            m_amc;    // Microdode PC
+	PAIR            m_ppc;    // previous program counter
+	bool            m_fp;
+	bool            m_jzdra_waiting;
+	u32             m_consts[0x10] = { 0x4, 0x2, 0x8, 0x1, 0x0, 0, 0, 0x66, 0, 0xc00000, 0xffffffff, 0x1f0000, 0x4000000, 0, 0, 0 };
+	PAIR            m_reg[0x20];
+	int             m_icount;
+	u32             m_op_size;
+
+	std::deque<u32> m_pcm_queue;
```

Comment:
You’ll need to change this to something else to support save states.  You can’t register a `std::deque` for save states.